### DISCUSSION TITLE
v2.17.9

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#5826] [General]
## What's Changed
* call livewire's getValidationAttributes() in form validation by @wychoong in https://github.com/filamentphp/filament/pull/5809
* Add missing translations for brazilian portuguese by @joaoprado in https://github.com/filamentphp/filament/pull/5815
* Fix unique validation docs link by @zepfietje in https://github.com/filamentphp/filament/pull/5814
* Fixed import by @eazyurk in https://github.com/filamentphp/filament/pull/5819
* how to access ownerRecord in RelationManagers by @tanthammar in https://github.com/filamentphp/filament/pull/5813
* fix: Nested repeater actions by @danharrin in https://github.com/filamentphp/filament/pull/5825

**Full Changelog**: https://github.com/filamentphp/filament/compare/v2.17.8...v2.17.9

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v2.17.9'>v2.17.9</a>.</em>

**Comment by diegobas** [comment]
Changes made in _packages/support/resources/views/components/dropdown/list/item.blade.php_ seem to break something. I get this error:

![image](https://user-images.githubusercontent.com/4599758/222366038-997c5ce8-6d35-4183-9c16-4eb932b27e2f.png)

**Reply by zepfietje** [reply]
@diegobas, thanks for reporting! This will be fixed as soon as possible. 👍

**Reply by danharrin** [reply]
It was fixed

**Reply by zepfietje** [reply]
Fixed in [v2.17.10](https://github.com/filamentphp/filament/releases/tag/v2.17.10).

**Comment by ismail-bouaichi** [comment]


**Reply by danharrin** [reply]
It's possible, but we're not gonna do your work for you. And this is not a place to ask questions either.
